skaffold.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: skaffold/v4beta11
+kind: Config
+metadata:
+  name: maintenance-operator
+
+profiles:
+  - name: operator
+    build:
+      local:
+        useBuildkit: true
+      artifacts:
+        - image: maintenance-operator-controller-manager
+          runtimeType: go
+          docker:
+            dockerfile: Dockerfile
+    manifests:
+      ## Deploy the manager.
+      kustomize:
+        paths:
+          - config/default
+
+    portForward:
+      - resourceType: deployment
+        resourceName: maintenance-operator-controller-manager
+        namespace: maintenance-operator-system
+        port: 56268
+        localPort: 8082
```

Comment:
Can you clarify how this portForward is used?

Reply:
thats for connecting remote debugger